tests/test_tensor.py

```diff
@@ -237,6 +237,14 @@ def test_tensor_tile(backend):
     tb = pyhf.tensorlib
     assert tb.tolist(tb.tile(tb.astensor(a), (2,))) == [1, 2, 3, 1, 2, 3]
 
+    a = [10, 20]
+    tb = pyhf.tensorlib
```

Comment:
As `tb = pyhf.tensorlib` already exists on lines 233 and 237 is this needed? If not, then I think 237 can also be removed.

```suggestion
```